### ISSUE TITLE
Switching to Peggy

### ISSUE THREAD

**Issue by ethan-ou** [fuhrmanator/GIFT-grammar-PEG.js#50]
Since the original PEG.js hasn't been maintained in a while, a [new fork of the project called Peggy](https://github.com/peggyjs/peggy) was created by the community. It appears to be a drop-in replacement, but will require a bit of testing to see whether previous PEG plugins are still compatible with this fork.

**Comment by fuhrmanator** [comment]
Yes, if it's providing added value I think it would be good to switch over. Updating means updating the downstream stuff, esp the GIFT quiz editor (Google Forms addon) and it's a lot of hoops. 

Google engineers have been updating some of the Quiz API feature requests. When an important release comes out (that would enable more GIFT conversion in Google Quiz) it might be a good time to move to Peggy.

Do you see any benefit of moving to Peggy for the VSCode suite of extensions at this time?

**Comment by ethan-ou** [comment]
At the moment there aren't any features that make it worth switching over on my end. But I'm all for backing a fork that's moving forward with the library.

This is definitely in the chore category/low priority, so I think the best time to move forward on this is the next time we make a major change to the parser. Happy to see some people picking up PEG.js again though!
